src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -56,11 +56,9 @@ import {
     generateScatterPlotDownloadData,
     getChartMetaDataType,
     getClinicalAttributeUniqueKey,
-    getClinicalAttributeUniqueKeyByDataTypeAttrId,
```

Comment:
Let's directly remove this method to make sure all places are removed.

Reply:
We can use ClinicalAttributeID as the uniqueKey going forward, but let's put that logic in the `getClinicalAttributeUniqueKey` so we can switch easily next time if the requirement changes

Reply:
its already removed